htdocs/inbox/compose.bml

```diff
@@ -203,7 +203,7 @@ body<=
         $reply_u = $msg->other_u;
         $reply_to = $reply_u->display_name;
         $disabled_to = 1;
-        $msg_subject = $msg->subject || "(no subject)";
+        $msg_subject = LJ::dhtml( $msg->subject ) || "(no subject)";
```

Comment:
@pinterface is correct - if you look at LJ/Message.pm, the difference between `subject` and `subject_raw` is that the former is wrapping the latter in an ehtml call and adding the `|| "(no subject)"` text.  So the better change here would be to simply change `subject` to `subject_raw`, instead of escaping a thing and then immediately unescaping it.

Thanks @kaberett for working on this and to everyone else for their reviews and comments.  ❤️